fiftyone/operators/store/clone.py

```
@@ -0,0 +1,170 @@
+    import) and skipped if unavailable, so a panel that isn't installed simply
+    isn't cloned.
+    """
+    names = list(_registered_store_names)
+
+    try:
+        from plugins.panels.similarity_search.constants import STORE_NAME
+
+        if STORE_NAME not in names:
+            names.append(STORE_NAME)
+    except Exception:
+        pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)